src/document-click.js

```diff
@@ -131,9 +131,17 @@ export function onDocumentElementClick_(e, viewport, history) {
   // to the custom protocol href.
   const isSafariIOS = platform.isIos() && platform.isSafari();
   const isEmbedded = win.parent && win.parent != win;
```

Comment:
Done.

Just noticed the isEmbedded check in the constructor. Thanks for pointing this out.